packages/data-store/src/migrations/postgres/1690925872592-CreateContacts.ts

```diff
@@ -0,0 +1,92 @@
+import { MigrationInterface, QueryRunner } from 'typeorm'
+
+export class CreateContacts1690925872592 implements MigrationInterface {
+  name = 'CreateContacts1690925872592'
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`ALTER TABLE "CorrelationIdentifier" DROP CONSTRAINT "FK_CorrelationIdentifier_identityId"`)
```

Comment:
As soon as you start merging you will see that in the old/first createContacts migrations I have added a first line 
```typescript
await enableUuidv4(queryRunner)
```

You will need that in here as well. This enables the extension `uid-ossp` on the Postgresql DB, which is needed for the uuid functions being called later on. This was the original error I was talking about 1 or 2 weeks ago.

Reply:
added to the migration